src/main/java/com/_up/megastore/data/model/Token.java

```diff
@@ -0,0 +1,27 @@
+package com._up.megastore.data.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Entity(name = "tokens")
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Data
+public class Token {
+    @Id
+    private UUID activationToken;
```

Comment:
Banco que la propia ID sea el token de activación. Ya que vamos a asignar un randomUUID() podríamos hacerlo directamente acá (y agregarle el final, por supuesto)
```suggestion
    private final UUID tokenId = UUID.randomUUID();
```

Reply:
En este caso, ¿cómo sería luego con la renovación de nuevos tokens?, crearíamos otro objeto o actualizaríamos su UUID, porque en el segundo caso, no podríamos si usamos final.

Reply:
Se debería generar un nuevo token